app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
```

Comment:
hidden is false by default, but if I want to create an animal with hidden=True I will get an error.